lib/logger.js

```diff
@@ -1,3 +1,5 @@
+/// <reference path="../typings/node/node.d.ts" />
```

Comment:
prefer to give node.d.ts to the compiler as an input, don't reference in your code